lib/gcs_signer.ex

```diff
@@ -23,27 +25,32 @@ defmodule GcsSigner do
 
   """
   @spec sign_url(
-    %{client_email: String.t, private_key: String.t},
-    String.t,
-    String.t,
-    sign_url_opts
-  ) :: String.t
+          %{client_email: String.t(), private_key: String.t()},
```

Comment:
Same with the change above - may I ask if there is a reason why indentation and method call format has changed here? I would prefer keeping the original indentation to match with other parts of our codebase.